### PR TITLE
Adjustment in MessageContext of Assistant_V2

### PR DESCRIPTION
**Pull request by AndreiLuis**
This is the most recommended for C #

For more information: [link](https://docs.microsoft.com/pt-br/dotnet/csharp/programming-guide/classes-and-structs/using-properties)

### CODE REVIEW

File: examples/IBM.Watson.Assistant.v2.Examples/ServiceExample.cs [hunk 1]
```diff
@@ -106,12 +107,15 @@ public void MessageWithContext()
...
                 apikey: "{apikey}");
 
-            AssistantService service = new AssistantService("2019-02-28", authenticator);
+            AssistantService service = new AssistantService("2020-04-01", authenticator);
             service.SetServiceUrl("{serviceUrl}");
 
             MessageContextSkills skills = new MessageContextSkills();
             MessageContextSkill skill = new MessageContextSkill();
-            skill.UserDefined.Add("account_number", "123456");
+            Dictionary<string, object> userDefinedDic = new Dictionary<string, object>();
```
Comment (mediumTaj) [inline examples/IBM.Watson.Assistant.v2.Examples/ServiceExample.cs:115]: Can you refactor this to `userDefinedDictionary`?

Comment (mediumTaj) [reply examples/IBM.Watson.Assistant.v2.Examples/ServiceExample.cs:115]: Also, can you sign the CLA?
https://cla-assistant.io/watson-developer-cloud/dotnet-standard-sdk?pullRequest=424

Comment (AndreiLuis) [reply examples/IBM.Watson.Assistant.v2.Examples/ServiceExample.cs:115]: Ohh okay, I'll do

**Pull request merged**